### DISCUSSION TITLE
Keep both Stable and Nightly on the same computer

### DISCUSSION THREAD

**Discussion by amitkot** [LunarVim/LunarVim#4180] [Q&A] [answered]
Is there a way to have both Stable and Nightly on the same computer?

The use case is I usually want to use the stable version, but currently NeoVim has a bug when using nvim-dap-ui that makes it close to unusable for some of my needs (https://github.com/rcarriga/nvim-dap-ui/issues/262).

I would love to have a LunarVim version that I can use with nightly nvim for these cases.

Thanks! 🙏

**Comment by LostInTheLogs** [comment, marked as answer]
you can probably use the current nvim nightly with latest lunarvim stable, but don't open issues if stuff doesn't work, you can also have a second lunarvim installation - just clone the lunarvim repo to ~/.config/lvim2 and start it with `NVIM_APPNAME="lvim2" nvim`
